scripts/remove

```diff
@@ -34,52 +34,124 @@ app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get $app domain)
 firmware_nonfree=$(ynh_app_setting_get $app firmware_nonfree)
+service_name=$(ynh_app_setting_get $app service_name)
+final_path=$(ynh_app_setting_get $app final_path)
+
+#=================================================
+# STANDARD REMOVE
+#=================================================
+# REMOVE SERVICE FROM ADMIN PANEL
+#=================================================
+
+# Remove a service from the admin panel, added by `yunohost service add`
+if yunohost service status | grep -q $service_name
```

Comment:
c.f. https://github.com/YunoHost/example_ynh/pull/73/files

I believe it should be more efficient to do : 
```suggestion
if yunohost service status $service_name >/dev/null 2>&1
```